ynr/apps/sopn_parsing/helpers/pdf_helpers.py

```diff
@@ -46,6 +46,18 @@ def document_heading_set(self):
         """
         return self.pages[1].get_page_heading_set()
 
+    @property
+    def intro_heading_set(self):
+        """
+        Takes the heading from the first or second page (if any) in the document and returns it as
+        a set
+        """
+
+        if len(self.pages) >= 2:
+            return self.pages[2].get_page_heading_set()
```

Comment:
Checking the second page should be the fallback rather than the default, because there might be some documents where the first page is fine and contains data, but has a blank second page or a page with just a few words, which would now raise the NoTextInDocumentError. I see the flow as:
- Check the first page document heading set as usual
- If it has less than 10 words, and we have more than 1 page, try checking the second page heading